tests/functional/ui/test_error_improvements.py

```
@@ -18,12 +22,65 @@
+    else:
+        suffix = pool_name.replace("ocs-storagecluster-cephfilesystem-", "")
+        try:
+            cf = OCP(kind="CephFilesystem", namespace=namespace)
+            cf_data = cf.get(resource_name="ocs-storagecluster-cephfilesystem")
+            data_pools = cf_data.get("spec", {}).get("dataPools", [])
+            for i, dp in enumerate(data_pools):
+                if dp.get("name") == suffix:
+                    cf.patch(
+                        resource_name="ocs-storagecluster-cephfilesystem",
+                        params=(
+                            f'[{{"op": "remove", ' f'"path": "/spec/dataPools/{i}"}}]'
```

Ruff (0.15.21)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)